<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomProcessDialog</class>
 <widget class="QDialog" name="CustomProcessDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>651</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="infoTab">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <attribute name="title">
       <string>Info</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="loadingLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="actionLabel">
         <property name="text">
          <string>fdfjdskkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="debugTab">
      <attribute name="title">
       <string>Debug</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPlainTextEdit" name="debugEdit">
         <property name="plainText">
          <string>Debug output : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="stdoutEdit">
         <property name="plainText">
          <string>stdout : 
</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="stderrEdit">
         <property name="plainText">
          <string>stderr : </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
